addons/binding/org.openhab.binding.edimax/ESH-INF/i18n/edimax.properties

```diff
@@ -0,0 +1,17 @@
+# binding
```

Comment:
Yes, if the texts are already in the thing-types.xml, there is no need for this property file.